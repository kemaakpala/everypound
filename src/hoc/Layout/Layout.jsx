import CSSBaseLine from '@material-ui/core/CssBaseline';
import Container from '@material-ui/core/Container';
import Header from '../../Components/Header/Header';
import Footer from '../../Components/Footer/Footer';

const Layout = ({ children }) => (
    <>
      <CSSBaseLine />
      <Container maxWidth="lg">
        <Header />
        <main>{children}</main>
        <Footer />
      </Container>
    </>
);

export default Layout;