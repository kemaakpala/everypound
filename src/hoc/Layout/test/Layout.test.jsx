import Layout from '../Layout';
import { render } from '@testing-library/react';

describe('<Layout/>', () => {
  it('matches snapshot', () => {
    const { firstChild } = render(<div><Layout>this is a test</Layout></div>)
    expect(firstChild).toMatchSnapshot();
  })
})