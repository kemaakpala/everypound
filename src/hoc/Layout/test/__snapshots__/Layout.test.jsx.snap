// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout/> matches snapshot 1`] = `undefined`;
